book/accessibility.md

```diff
@@ -731,36 +739,108 @@ indicated.)
 Keyboard navigation
 ===================
 
-Our browser is currently mouse-only.^[Except for scrolling, which is
-keyboard-only.] This is problematic, because there are a number of reasons why
-users might want to use the keyboard to interact instead. Reasons such as
-physical inability, injury to the hand or arm from too much movement, or
-being a power user that finds keyboards more efficient than mice.
+Right now, most browser features are triggered using the
+mouse,^[Except for scrolling, which is keyboard-only.] which is a
+problem for users with injuries or disabilities in their hand---and
+also a problem for power users that prefer their keyboards. So ideally
+every browser feature should be accessible via the keyboard as well as
+the mouse. That includes both browser chrome interactions like going
+back, typing a URL, or quitting the browser, and also web page
+interactions such as submitting forms, typing in text areas, and
+navigating links.
+
+Let's start with the browser chrome, since it's easiest. Here, we need
+to allow the user to go back, to type in the address bar, and to
+create and cycle through tabs, all with the keyboard. We'll also add a
+keyboard shortcut for quitting the browser.[^one-more] Let's make all
+these shortcuts use the `Ctrl` modifier key so they don't interfere
+with normal typing: `Ctrl-Left` to go back, `Ctrl-L` to type in the
```

Comment:
Technically the code uses the lowercase letter, change the text to match?

Reply:
We should standardize how we write keyboard shortcuts. But, for now, done.